css/jquery.uls.mobile.css

```diff
@@ -40,15 +40,15 @@
 		float: none;
 	}
 
-	.uls-mobile [class*="column"] + [class*="column"]:last-child {
+	.uls-mobile [ class*='column' ] + [ class*='column' ]:last-child {
```

Comment:
This looks weird. We don't do this kind of spacing in CSS in CX for example. Where is it coming from.

Reply:
Comes from https://github.com/wikimedia/stylelint-config-wikimedia/commit/a65028503ef0bab5dc99dc24c3768f582d46d1c2#diff-168726dbe96b3ce427e7fedce31bb0bc

Looks like we did not update to this version of stylelint in CX?